### PR TITLE
basic-installation.md, advanced-installation.md

### PR DESCRIPTION
**Pull request by nikolokol**
## Description of improvements

Any specific reason for this change? If a recent change in the MODX core caused the change, please include a link to the relevant pull request. 

## Affected versions

Please make sure your pull request targets the **lowest** affected version. Mark other relevant versions with an X below.

- [x] 2.x (current)
- [x] 3.x

## Relevant issues

If available, provide the link or ID of relevant issues so that they can be closed once your improvement has been merged.

### CODE REVIEW

File: ru/getting-started/installation/basic-installation.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      - [Существующий динамически управляемый сайт с другой CMS](#BasicInstallation-ExistingOtherCMSor%EF%BB%BFDynamicallyDrivenSite)
+  - [Установка во временный каталог на время переноса DNS](#BasicInstallation-PreDNSTransferInstallationtoTemporaryDirectory)
+- [Установка MODX Revolution](#BasicInstallation-InstallingMODXRevolution)
+- [Варианты установки](#BasicInstallation-InstallOptions)
+- [Параметры базы данных](#BasicInstallation-DatabaseOptions)
+  - [Сопоставление и Кодировка](#BasicInstallation-CollationsandCharsets)
+  - [Создание Администратора по умолчанию](#BasicInstallation-CreatinganAdministratorUser)
+- [Проверка перед установкой](#BasicInstallation-PreInstallationChecks)
+- [Резюме после установки](#BasicInstallation-PostInstallationSummary)
+- [Дополнительная информация](#BasicInstallation-AdditionalInfo)
```
Comment (Ibochkarev) [inline ru/getting-started/installation/basic-installation.md]: Нужно выровнять по левому краю, так как все поехало.

File: ru/getting-started/installation/advanced-installation.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  4. [Установка на сервер под управлением ModSecurity](getting-started/installation/basic-installation/installation-on-a-server-running-modsecurity)
+  5. [MODX и Suhosin](getting-started/installation/basic-installation/modx-and-suhosin)
+  6. [Конфигурация сервера Nginx](getting-started/installation/basic-installation/nginx-server-config)
+2. [Расширенная установка](getting-started/installation/advanced-installation)
+3. [Установка Git](getting-started/installation/git-installation)
+4. [Установка из командной строки](getting-started/installation/command-line-installation)
+  1. [Файл Xml с параметрами установки](getting-started/installation/command-line-installation/the-setup-config-xml-file)
+5. [Устранение неполадок при установке](getting-started/installation/troubleshooting-installation)
+6. [Успешная установка, что делать дальше?](getting-started/installation/successful-installation,-now-what-do-i-do)
+7. [Использование MODx Revolution с SVN](getting-started/installation/using-modx-revolution-from-svn)
```
Comment (Ibochkarev) [inline ru/getting-started/installation/advanced-installation.md]: Нужно выравнять по левому краю и восстановить порядок цифр

File: ru/getting-started/installation/advanced-installation.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+---
+title: "Расширенная установка"
+translation: "getting-started/installation/advanced-installation"
+---
+
+- [Предварительные шаги по установке] (#AdvancedInstallation-InstallationPreSteps)
```
Comment (Ibochkarev) [inline ru/getting-started/installation/advanced-installation.md]: Выравнять по левому краю.

File: ru/getting-started/installation/basic-installation.md (hunk 1 above)
Comment (nikolokol) [reply ru/getting-started/installation/basic-installation.md]: Не понял, это же разные уровни заголовков, как здесь [https://github.com/nikolokol/Docs-MODX/edit/2.x/en/getting-started/installation/basic-installation.md](url)

Comment (Ibochkarev) [reply ru/getting-started/installation/basic-installation.md]: @nikolokol https://github.com/Ibochkarev/Docs-MODX/blob/95ddf5b351c7d3b90996d41a971a7d43731c4075/ru/getting-started/installation/advanced-installation.md

docs-modxadvanced-installation md at 95ddf5b351c7d3b90996d41a971a7d43731c4075 ibochkarevdocs-modx 2019-03-06 11-02-29 1

Так будет понятнее. Разметка поехала

Comment (nikolokol) [reply ru/getting-started/installation/basic-installation.md]: Поменял [https://github.com/Ibochkarev/Docs-MODX/blob/59b770d731a195552ef9df6db2c46e872012b538/ru/getting-started/installation/advanced-installation.md](url)

**Pull request merged**